prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -45,7 +45,7 @@ presubmits: # runs on PRs
           path_alias: github.com/kyma-project/test-infra
       spec:
         containers:
-          - image: eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
+          - image: eu.gcr.io/kyma-project/prow/gke-integration:0.0.1
```

Comment:
regarding image name, I also don't have a good idea how to name it, but maybe bootstrap-helm will be a better option?